src/transformers/models/dpt/image_processing_dpt.py

```diff
@@ -106,7 +106,7 @@ class DPTImageProcessor(BaseImageProcessor):
         ensure_multiple_of (`int`, *optional*, defaults to 1):
             If `do_resize` is `True`, the image is resized to a size that is a multiple of this value. Can be overidden
             by `ensure_multiple_of` in `preprocess`.
-        resample (`PILImageResampling`, *optional*, defaults to `PILImageResampling.BILINEAR`):
+        resample (`PILImageResampling`, *optional*, defaults to `PILImageResampling.BICUBIC`):
```

Comment:
This is a (slight) breaking change to make sure the same interpolation method is used as in the [original implementation](https://github.com/isl-org/MiDaS/blob/bdc4ed64c095e026dc0a2f17cabb14d58263decb/midas/model_loader.py#L221). However, Pillow's BICUBIC method does not 100% match the one of OpenCV :/ cc @amyeroberts

Reply:
There's never 1:1 correspondence 😢 

This is OK as saved models will have the resampling filter saved in the preprocessor config, and as you say it brings it in line with the original